fuzz/make_corpus.py

```diff
@@ -0,0 +1,59 @@
+import json
+import argparse
```

Comment:
Please reorder. (Also, we probably need a Python linter if we start using Python.)

Reply:
There is, but it didn't detect this somehow :(